addons/website/static/src/components/dialog/dialog.scss

```diff
@@ -2,4 +2,8 @@
     label {
         font-weight: $font-weight-bold;
     }
+
+    .form-group {
+        margin-bottom: 1rem;
+    }
```

Comment:
The form-group class apparently does not exist anymore in BS5... and apparently the class was kept everywhere in the codebase anyway :( Migrating to BS5 does not mean restoring its old rules case by case... :grimacing: Here what you have to do is properly migrate to BS5, not restore BS4 rules to make it work :/